### ISSUE TITLE
Slow boot

### ISSUE THREAD

**Issue by slaff** [raburton/rboot#8]
When using rBoot the booting time is roughly 7 times higher.

I used the Basic_Blink example in Sming. 
Without rBoot, the boot time is ~ 300 ms
With rBoot and with/or without RTC_ENABLED the boot time is ~2300  ms

Any ideas why is that happening ?

**Comment by raburton** [comment]
The rBoot code has a 2 second delay to allow you to connect a terminal
before the messages are printed. This should be commented out by default,
but I could have accidentally committed it uncommented? rBoot doesn't
normally slow the boot by any significant amount and the fact it seems to
have done so here by exactly 2 seconds makes me suspect that deliberate
delay is in effect.

Richard.

**Comment by slaff** [comment]
Got it! In rboot.c: 

https://github.com/SmingHub/Sming/blob/develop/Sming/rboot/rboot.c#L220

```
	// delay to slow boot (help see messages when debugging)
	ets_delay_us(2000000);
```

Will you change this or I should submit a PR?

**Comment by raburton** [comment]
fixed in 4cf6132f6ee624318c04b2a8d9e37da64b7c283c
